deploy/charts/rawfile-csi/values.yaml

```diff
@@ -12,6 +12,10 @@ defaults: &defaults
     requests:
       cpu: 10m
       memory: 100Mi
+  rawfile_dir: /rawfile
```

Comment:
Please change this to `/var/csi/rawfile` to keep it backward-compatible.

Reply:
sure, I will change it later

Reply:
hello, @SeMeKh I've updated this to `/var/csi/rawfile`
can you please check it again?
thank you